### PR TITLE
docs: add remix example to docs

### PR DESCRIPTION
**Pull request by shawshankkumar**
Add a remix example on how to work with formbricks. I made a simple feedback form and linked it, seems to be working well.

image

### CODE REVIEW

File: remix/app/root.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+} from "@remix-run/react";
+import formbricks from "@formbricks/js";
+
+export const links: LinksFunction = () => [
+  ...(cssBundleHref ? [{ rel: "stylesheet", href: cssBundleHref }] : []),
+];
+
+if (typeof window !== "undefined") {
+  formbricks.init({
+    environmentId: "clm7mda0f045tjs0gwrm1a8gc",
```
Comment (ShubhamPalriwala) [inline remix/app/root.tsx:19]: hey, do you mind replacing this with a placeholder like `<environment-id>` to prevent spam submissions in your environment!

**Review by ShubhamPalriwala — COMMENTED** [review]
Thanks a lot @shawshankkumar for this 🚀 ! I'll merge this as soon as you can remove your environment ID as mentioned below! 🙏🏼

**Comment by shawshankkumar** [comment]
Sure sure. I was going to delete it anyway my bad!

**Comment by shawshankkumar** [comment]
@ShubhamPalriwala should be fixed now!

**Review by ShubhamPalriwala — APPROVED** [review]

**Pull request merged**